src/components/footer/index.tsx

```diff
@@ -131,17 +135,19 @@ function Footer({
 						</li>
 					)
 				})}
-				<li>
+				<li className={classNames(s.actionDesktop, s.linkListItem)}>
 					<ButtonLink
 						text="Give Feedback"
 						href={FEEDBACK_FORM_URL}
 						color="secondary"
 						size="small"
 						icon={<IconPencilTool16 />}
 						opensInNewTab={true}
-						className={classNames(s.feedbackButton, s.desktop)}
 					/>
 				</li>
+				<li className={classNames(s.actionDesktop, s.linkListItem)}>
+					<ThemeSelect />
```

Comment:
Probably need to add an aria label since there isn't a visible label

Reply:
Good call!